### ISSUE TITLE
About training detial

### ISSUE THREAD

**Issue by Limingxing00** [z-x-yang/AOT#1]
Hi, Zongxin,

Thank you for the nice work.   I am concerned about the training detail.

1. How many models do you use in Table 1?   Davis valid/test share one and Youtube 18/19 share one?   
2. You say "For main training, the training steps are 100,000 for YouTube-VOS or 50,000 for DAVIS" and what is the iteration of total training?  Because I think "training steps" is a middle step of adjusting the learning rate.

**Comment by z-x-yang** [comment]
1. Yes. Davis valid/test share one and Youtube 18/19 share one.

2. Pretraining on static images: 100,000 steps (about 1.1 days for AOT-L with 4 Tesla V100 GPU).
Main training on YouTube-VOS: 100,000 steps (about 1.1 days for AOT-L with 4 Tesla V100 GPU).
Main training on DAVIS or finetune YouTube-VOS models on DAVIS: 50,000 steps (about 0.6 days for AOT-L with 4 Tesla V100 GPU).

**Comment by Limingxing00** [comment]
Thank you for your quick reply!
